code/modules/jobs/job_types/roguetown/adventurer/types/antag/foresworn.dm

```diff
@@ -1,4 +1,4 @@
-/datum/advclass/foresworn //Strength class, starts with axe or flails and medium armor training
+/datum/advclass/foresworn //Tanky class, starts with good con/end and spears or sword/shield
```

Comment:
solid whoops on my part. That comments was copy pasted from brigand. Foresworn was designed more of a generalist, then a strength class.